beacon-chain/p2p/monitoring.go

```diff
@@ -61,16 +61,20 @@ var (
 			"until a subnet peer can be found.",
 	})
 	blobSidecarBroadcasts = promauto.NewCounter(prometheus.CounterOpts{
-		Name: "p2p_blob_sidecar_committee_broadcasts",
-		Help: "The number of blob sidecar messages that were broadcast with no peer on.",
+		Name: "p2p_blob_sidecar_broadcasts",
+		Help: "The number of blob sidecar messages that were broadcasted with no peer on.",
```

Comment:
There is multiple issues with the code currently in the `peerDAS` (and `develop`) branches:
- The word `committee` has nothing to do here, as you fixed. (Changing this name is a breaking change, as it will possibly break some monitoring or alerting. So we'll need to add a dedicated comment in the next release note. But that's OK to change it.)
- The behaviours of these metrics is strange. The expected behaviour is:
    - For `p2p_blob_sidecar_attempted_broadcasts` (and its equivalent for data column): To be increased by 1 at every attempt (successful or not)
    - For `p2p_blob_sidecar_broadcasts` (and its equivalent for data column): To be increase by 1 at every successful broadcast.

Helps need to be updated accordingly.

If you wish to apply these changes in this PR it would be great. Else, I'll do it in an other PR.

Reply:
I'll spin up another PR to fix this and upstream it to `develop`, no reason to let it stay in PeerDAS and wait for it to be merged in one giant step (which would definitely complicate things)

Reply:
As for this current PR, I'll revert changes related to blob metrics

Reply:
Nice, thanks.

Also, I just had to rebase the `peerDAS` branch on top of `develop`. It messed up your branches, sorry for that.
An easy way to fix that is to:
- Reset your branches on `peerDAS`
- Cherry pick the commit you added
- Push force

For information, we decided first to create a long running `peerDAS` branch because peerDAS was not intended at all to be part of pectra, and we wanted to have something quickly working for the interop last month. (The `peerDAS`branch was not intended to be merged into `develop`.

It turned out that teams made some good progresses on peerDAS, so peerDAS is now possibly going into pectra. (And so we have to clean our long running `peerDAS` branch before merging into `develop`.)

Reply:
Rebased, wondering if there's any channels that we can DM instead of talking over github?

For context, I'm from Base team and we're more than thrilled to have PeerDAS included in Pectra as this increases blob capability for everybody and certainly for us that relieves concerns about our near term scaling ceilings on DA.

We're actively contributing in various aspects to try to make PeerDAS as mature as possible for inclusion. Would love to connect and collaborate more closely